demosplan/DemosPlanCoreBundle/Logic/User/UserHandler.php

```diff
@@ -1802,11 +1802,16 @@ public function wipeOrganisationData(string $organisationId)
                 && $successfulDeletedMasterToebs
                 && $successfulDeletedMasterToebMail
                 && $successfulDeletedDepartments) {
-                return $this->orgaService->wipeOrganisation($organisationId);
+                $result = $this->orgaService->wipeOrganisation($organisationId);
+                if ($result instanceof Orga) {
+                    return $result;
```

Comment:
I think if errors is at this point empty and one of the conditions $successfulRemovedSettings, $successfulDeletedAddresses, $successfulDeletedMasterToebs, $successfulDeletedMasterToebs, $successfulDeletedMasterToebMail and $successfulDeletedDepartments is false, then the return is missed.
The next return returns $errrors, and then $errors could be empty, or not?

Reply:
done in [fa1dd56](https://github.com/demos-europe/demosplan-core/pull/2681/commits/fa1dd566f130d0728da1f338892fca8e96ff2612)